app/scripts/templates/global_header/global_header.html

```diff
@@ -1,4 +1,4 @@
-<div class="row-inner l-flex-horizontal">
+<div class="row-inner-padded l-flex-horizontal">
```

Comment:
@nchapman: not sure how we'd do that since the flex classes are applied in various non-row contexts and, not all rows are necessarily flexed, but yeah, I'm sure we can figure this one out.